src/components/Alert.jsx

```diff
@@ -0,0 +1,54 @@
+export function Alert({ crud, setDisplayAlert }) {
```

Comment:
I've no doubt all this does something, but I'm struggling to work out what it does? It called Alert, but when I click vote up or down on your app, I don't appear to get any alerts? The vote count just changes +1 or -1 depending on which button I press. Is this `Alert` component serving a purpose?